### ISSUE TITLE
Uncaught TypeError: htmlspecialchars() in NavigationTree

### ISSUE THREAD

**Issue by b3acon** [phpmyadmin/phpmyadmin#15540]
### Describe the bug

after copy a db over options. the 5.0.0-alpha1 locked me out with this fatal.error.
used the default options to copy it.

### To Reproduce

1. Go to 'select a database'
2. Click on 'options'
2. Fill out 'new databse name'
4. Click on 'copy databse'

```
Fatal error: Uncaught TypeError: htmlspecialchars() expects parameter 1 to be string, int given in /libraries/classes/Navigation/NavigationTree.php:787 Stack trace: 
#0 /libraries/classes/Navigation/NavigationTree.php(787): htmlspecialchars(125612) 
#1 /libraries/classes/Navigation/NavigationTree.php(856): PhpMyAdmin\Navigation\NavigationTree->groupNode(Object(PhpMyAdmin\Navigation\Nodes\Node)) 
#2 /libraries/classes/Navigation/NavigationTree.php(856): PhpMyAdmin\Navigation\NavigationTree->groupNode(Object(PhpMyAdmin\Navigation\Nodes\Node))
#3 /libraries/classes/Navigation/NavigationTree.php(690): PhpMyAdmin\Navigation\NavigationTree->groupNode(Object(PhpMyAdmin\Navigation\Nodes\NodeDatabaseContainer)) 
#4 /rp-h in /libraries/classes/Navigation/NavigationTree.php on line 787
```
### Server configuration

- Operating system: gentoo
- Web server: apache
- Database version: 5.7
- PHP version: 7.3.2
- phpMyAdmin version: 5.0.0-alpha1

### Client configuration

- Browser: chrome
- Operating system: osx

**Comment by williamdes** [comment]
Reproduced using this setup, the issue only occurs when you have groups
![db groups](https://user-images.githubusercontent.com/7784660/67904696-71c22000-fb6f-11e9-9502-e206f47b32c2.png)

**Comment by williamdes** [comment]
Fixed by 128d3a09a0392a5a0a6c4e6ea9aefcd8ddf9efca
@b3acon please apply the fix and you will be okay until next 5.0 release, I also included the fix in 4.9 as explained in ab0b6e248a9c77c782f1274a63ea426d40cc300d
